conf/application.conf

```diff
@@ -360,7 +360,7 @@ urls {
 
     ei = "/employment-intermediary-report/not-enrolled"
     ated = "/ated/home?callerId=bta"
-    lbtScotland = "https://www.revenue.scot/land-buildings-transaction-tax/forms"
+    lbtScotland = "https://revenue.scot/taxes/land-buildings-transaction-tax/paper-lbtt-forms"
```

Comment:
Why has this been changed? The app-config values are still https://www.revenue.scot/land-buildings-transaction-tax/forms